blog/content/post/2019-11-22-Websummit-2019.md

```diff
@@ -0,0 +1,107 @@
+---
+title: '3mdeb on Web Summit 2019'
+abstract: 'Web Summit 2019 - the biggest tech event in the World.
+          Take a short brake and read about our journey and impressions.'
+cover: /covers/web_summit_1.jpg
+author: lukasz.wcislo
+layout: post
+published: true
+date: 2019-11-22
+archives: "2019"
+
+tags:
+  - Web Summit
+  - Conference
+categories:
+  - Miscellaneous
+
+---
+
+Founded in 2009, originally held in Dublin until 2016, when it moved permanently
+to Lisbon, Web Summit is considered to be the largest tech event in the world.
+Until 2019 there was no representation of 3mdeb on Web Summit, but now we've
+decided, that it is the right time to send one.
+
+"Tech event" is the keyword that turns all of us on and makes us travel over
+thousand kilometers, but after the struggle, I was the one who becomes the first
+official 3mdeb representative on that event. Full of hope and curiosity, with
+a great financial and organizational support provided by
+[Pomeranian Export Broker](http://www.brokereksportowy.pl), at Sunday, November
```

Comment:
@lwcislo please do not forget to send that to them or ping them on social media, this should be redirected to Staszek. @artur-rs cc

Reply:
Ok.